synthetics/test_schema.go

```diff
@@ -335,55 +345,60 @@ func makeTestMonitoringSettingsSchema(mode schemaMode) *schema.Schema {
 }
 
 func makeTestPingSchema(mode schemaMode) *schema.Schema {
-	return makeNestedObjectSchema(mode, map[string]*schema.Schema{
+	return makeNestedObjectSchemaOptional(mode, map[string]*schema.Schema{
 		"period": {
 			Type:     schema.TypeFloat,
-			Optional: optionalOnCreate(mode),
+			Optional: true,
```

Comment:
`period` is required on create.

Reply:
Addressed: a2e73bcd0db6697b5392fdb69a572f74228bc594

Reply:
Done